src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -307,7 +308,8 @@ const styles = StyleSheet.create({
   noDataFoundText: {
     alignSelf: 'center',
     justifyContent: 'center',
-    fontFamily: fonts.inter[600]
+    fontFamily: fonts.inter[600],
+    color: COLORS.white2
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes here involve setting the font family and color for the `noDataFoundText` style. Ensure that the chosen color provides sufficient contrast against its background for readability. Also, confirm that the specified font family (`fonts.inter[600]`) is loaded and available in the application, as missing fonts can lead to unexpected UI behavior.
```diff
+    fontFamily: fonts.inter[600],
+    color: COLORS.white2
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->